### PR TITLE
Direct readers to the docs for .NET Standard

### PR DESCRIPTION
**Pull request by richlander**

### CODE REVIEW

File: Documentation/architecture/net-platform-standard.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ﻿# .NET Platform Standard
 
+> The .NET Platform Standard concept has been repaced by the .NET Standard Library. Check out the [.NET Standard Library](https://docs.microsoft.com/dotnet/articles/standard/library) topic in the [.NET Documentation](https://docs.microsoft.com/dotnet). It provides a good first view into .NET Standard Library.
```
Comment (akoeplinger) [inline Documentation/architecture/net-platform-standard.md]: ~~repaced~~ replaced

File: Documentation/architecture/net-platform-standard.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 ﻿# .NET Platform Standard
 
+> The .NET Platform Standard concept has been replaced by the .NET Standard Library. Check out the [.NET Standard Library](https://docs.microsoft.com/dotnet/articles/standard/library) topic in the [.NET Documentation](https://docs.microsoft.com/dotnet). It provides a good first view into .NET Standard Library.
```
Comment (stephentoub) [inline Documentation/architecture/net-platform-standard.md]: If .NET Platform Standard has been deprecated in favor of .NET Standard Library, should this whole page just be deleted?

**Comment by stephentoub** [comment]
cc: @ericstj

File: Documentation/architecture/net-platform-standard.md (hunk 2 above)
Comment (ericstj) [inline Documentation/architecture/net-platform-standard.md]: I'm under the impression they are the same thing its just a name change to reduce concept count/confusion.  @richlander is that correct?  I've never understood what distinction we were trying to make with this term, see https://github.com/dotnet/corefx/pull/5729#discussion_r50991776.
I don't think it's right to say "replaced".  I would just change this doc to say its about .NET Standard library, remove any overlapping content, and point users to the main docs as "first reading".

**Comment by richlander** [comment]
"Platform Standard" and "Standard Library" are not the same concepts. Two links that demonstrate that:

- https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md#relationship-to-platforms
- https://github.com/dotnet/corefx/blob/2c52afa760bd512d8513549558d1b68ae39e1e6c/Documentation/architecture/net-platform-standard.md#list-of-proposed-standard-platform-contracts-from-corefx-tentative

There are two major issues with the document:

- The doc is/was a proposal and now we've made something and it differs from what the document describes.
- The doc describes many topics, some of which are tangential to the primary topic.

There are lots of links to this topic, so we cannot reasonably delete it. I propose we delete the content and just keep the "please look here" content. If folks want to retain content they think is important, we can PR it into core-docs. Most of it shouldn't be here anymore.

**Comment by ericstj** [comment]
I'm ok with deleting it, just need a home for the tables.  One of which I have build automation for generating.

**Comment by richlander** [comment]
Which tables do you generate? I included some of the tables in core-docs already?

**Comment by richlander** [comment]
I merged the initial text while we're discussing the final plan.

**Comment by ericstj** [comment]
This table is generated: https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md#list-of-net-corefx-apis-and-their-associated-net-platform-standard-version

**Pull request merged**